man/pam_systemd.xml

```diff
@@ -159,6 +159,20 @@
           </tgroup>
         </table>
 
+        <para>If no session class is specified via either the PAM module option, nor via the
+        <varname>$XDG_SESSION_CLASS</varname> environment variable, the class is automatically chosen, depending on
```

Comment:
Grammar: "if no session … is specified via … or via …". "nor" is not used like this.